src/main/java/gov/cms/madie/madiefhirservice/validators/CustomRemoteTerminologyServiceValidationSupport.java

```diff
@@ -138,6 +138,21 @@ public String getErrorMessage(String errorCode, Object... theParams) {
   public IBaseResource fetchValueSet(String theValueSetUrl) {
     // disable code value set fetch for remote terminology service because it appears that VSAC
     // doesn't support value set search with summary mode
-    return null;
+    // Added cases it calls for cts valuesets
+    if (StringUtils.isBlank(theValueSetUrl)
+        || !StringUtils.contains(theValueSetUrl, "cts.nlm.nih.gov")) {
+      return null;
+    } else {
+      Class<? extends IBaseBundle> bundleType =
+          this.myCtx.getResourceDefinition("Bundle").getImplementingClass(IBaseBundle.class);
+      IQuery<IBaseBundle> valueSetQuery =
+          client
+              .search()
+              .forResource("ValueSet")
+              .where(CodeSystem.URL.matches().value(theValueSetUrl));
```

Comment:
Although this works because CodeSystem.URL.matches() and ValueSet.URL.matches() uses same underlying method, maybe we can change it to ValueSet.URL because this is ValueSet specific
```suggestion
              .where(ValueSet.URL.matches().value(theValueSetUrl));
```